src/components/shapes/attributes.js

```diff
@@ -169,7 +169,28 @@ module.exports = templatedArray('shape', {
             'See `type` and `xsizemode` for more info.'
         ].join(' ')
     },
-
+    x0shift: {
+        valType: 'number',
+        dflt: 0,
+        min: -1,
+        max: 1,
+        editType: 'calc',
+        description: [
+            'Only relevant if `xref` is a (multi-)category axes. Shifts `x0` by a fraction of the',
+            'reference unit. E.g. -0.5 corresponds to the start of the category and 0.5 corresponds to the end of the category.'
```

Comment:
Shifts `x0` away from the centre of the category when `xref` is a "category" or "multicategory" axis. 
-0.5 corresponds to the start of the category and 0.5 corresponds to the end of the category.

What do you think of this @archmoj. Is it technically correct?

Reply:
@LiamConnors this is correct.
@my-tien Please adjust the descriptions and update the schema and I think we should be good to merge this PR.
Thank you!

Reply:
Thanks for turning it into something comprehensible :)
I just changed this part to use asterisk like you do in other descriptions:
> \*category\* or \*multicategory\* axis